common/source/docs/common-pocketbeagle-2.rst

```diff
@@ -0,0 +1,291 @@
+.. _common-pocketbeagle2:
+
+===============
+PocketBeagle 2
+===============
+
+`PocketBeagle 2 <https://www.beagleboard.org/boards/pocketbeagle-2>`__ (PB2) is an upgraded version of the popular PocketBeagle, designed as an ultra-compact, low-cost, and powerful single-board computer (SBC). Targeted at developers, students, and hobbyists, PocketBeagle 2 retains the simplicity and flexibility of its predecessor while delivering enhanced performance and expanded features to support modern development needs. PocketBeagle 2 is ideal for creating IoT devices, robotics projects, and educational applications. Its small form factor and low power consumption make it a versatile platform for embedded development, whether prototyping or deploying at scale.
+
+PocketBeagle 2 is based on Texas Instruments AM6254 SoC. Its multiple A53 cores can provide higher performance than the classic PocketBeagle. The new design comes with pre-soldered headers, a 3-pin JST-SH 1.00mm UART debug port, a USB-C port, Texas Instruments MSPM0L1105 Cortex-M0+ MCU for ADC, 512MB RAM, and a LiPo Battery charger.
+
+.. note::
+
+   For now, the only supported version is Rev A0 shipped with AM6232 (dual-core A53, no GPU).
+
+.. image:: ../../../images/pocketbeagle-2-front.jpg
+    :target: ../_images/pocketbeagle-2-front.jpg
+    :width: 800 px
+
+.. image:: ../../../images/pocketbeagle-2-back.jpg
+    :target: ../_images/pocketbeagle-2-back.jpg
+    :width: 800 px
+
+Specifications
+==============
+
+-  **Processor**
+
+   -  Texas Instruments AM6232 SoC
+   -  Multicore 64-bit Arm Cortex-A53 microprocessor subsystem at up to 1.4 GHz
+   -  Each A53 Core has 32KB L1 DCache with SECDED ECC and 32KB L1 ICache with Parity protection
+   -  Single-core Arm® Cortex®-M4F MCU at up to 400MHz
+   -  Dual-core Programmable Real-Time Unit Subystem (PRUSS) running up to 333 MHz
+   -  512MB LPDDR4 3200MHz
+   -  4GB 8-bit eMMC flash
+
+-  **OS**
+
+   -  Linux (Debian)
+
+.. note::
+
+   For now, the only supported version is Rev A0 shipped with AM6232 (dual-core A53, no GPU).
```

Comment:
Please test with the current production version (rev A1) that has quad-core and GPU, though GPU wouldn't have much use in this application. If you need a board for testing, please reach out.

Reply:
https://www.beagleboard.org/boards/pocketbeagle-2 has links to distributors. I'll privately provide you the link to the board-request form for people doing open source work.